python/vowpalwabbit/sklearn_vw.py

```diff
@@ -521,12 +618,14 @@ def predict(self, X):
         """Predict class labels for samples in X.
 
         Parameters
-        ----------
+        ==========
```

Comment:
Why change these to `=`, it looks like they are `-` in the numpy styleguide? https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard

Reply:
I have used it because when we use Sphinx for creating auto docs it treats them as headings and highlights them.
[http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html](http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html)